src/controller/java/src/chip/devicecontroller/ChipDeviceController.java

```diff
@@ -175,6 +175,14 @@ public void pairDeviceWithAddress(
         deviceControllerPtr, deviceId, address, port, discriminator, pinCode, csrNonce);
   }
 
+  public void pairDeviceWithCode(
```

Comment:
Please add doxygen for this public API, thanks

Reply:
I added.